src/components/ChangeDateModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Dialog, Button } from "@statisticsnorway/ssb-component-library";
+import { useTranslation } from "react-i18next";
+import "../css/DateModal.css";
+
+
+const ChangeDateModal = ({ isShowing, dismiss, submit }) => {
+    const {t} = useTranslation();
+
```

Comment:
This component is going to change AppContext, because it communicates with user.
It has to have access to the AppContext.subset.dispatch actions.
Is looks perfect to put an instance of useChangeDateModal() here, or just make the content of this hook to be a part of Modal's business logic.